.gitignore

```diff
@@ -27,6 +27,3 @@ Thumbs.db
 Carthage/
 
 Frameworks/
-
-.swiftlint-report.json
```

Comment:
I think it's better to add it to the ignore. Because the user may forget to remove them, and the gitignore is there to avoid those mistakes and avoid having to remove files all the time and make it easier for you. (at least that's what I think)